src/pages/Admin.jsx

```diff
@@ -118,9 +118,9 @@ const Admin = () => {
     }
 
     (async () => {
-        setMaxNftState( await parseInt(formatEther( await mContract.maxNFTCount())));
+        setMaxNftState( await parseInt( await mContract.maxNFTCount()));
```

Comment:
@antarpreet11  I ended up making these just INT as the contract does not use WEI here. sorry for the unclear directions